roles/jupyterhub_host/tasks/main.yml

```diff
@@ -30,3 +30,15 @@
         - "{{ docker_tls_path }}:{{ docker_tls_path }}"
   tags:
     - docker-rebuild
+
+- name: get a jupyterhub api token
+  command: docker exec jupyterhub jupyterhub token root
```

Comment:
I'm not sure `root` can be trusted to exist. The current user should only be added to the list if the admin list is empty. I think this also may require a `-f /srv/jupyterhub/jupyterhub_config.py` now that jupyter/jupyterhub#145 is merged.